shared/lib/custom_dc_util.py

```diff
@@ -27,6 +30,15 @@ def get_user_data_path() -> str:
   return os.environ.get('USER_DATA_PATH', '')
 
 
+def get_topic_cache_path() -> str:
```

Comment:
This and others should probably have `custom` in their name?

Reply:
Done.